thumbor/loaders/http_loader.py

```diff
@@ -72,23 +74,22 @@ def validate(context, url, normalize_url_func=_normalize_url):
 
 def return_contents(response, url, context, req_start=None):
     res = urlparse(url)
+    netloc = res.netloc.replace(".", "_")
+    code = response.code
+
     if req_start:
         finish = datetime.datetime.now()
         context.metrics.timing(
-            "original_image.fetch.{0}.{1}".format(
-                response.code, res.netloc.replace(".", "_")
-            ),
+            f"original_image.fetch.{code}.{netloc}",
             (finish - req_start).total_seconds() * 1000,
         )
         context.metrics.incr(
-            "original_image.fetch.{0}.{1}".format(response.code, res.netloc.replace(".", "_"))
+            f"original_image.fetch.{code}.{netloc}",
         )
 
     result = LoaderResult()
     context.metrics.incr("original_image.status." + str(response.code))
```

Comment:
Replace `response.code` with previously declared `code`

Reply:
Not really. Fixed! 👍